example.output

```diff
@@ -1,9 +1,9 @@
 Mapping for region ap-northeast-1
-+-----------------+---+---+---+
-| xxxxxxxxxxxxx   | a | c | d |
-| yyyyyyyyyyyyyyy | a | c | d |
-| zzzzzzzzzzzzz   | a | c | d |
-+-----------------+---+---+---+
++-----------------+---+---+---+---+
+| xxxxxxxxxxxxx   | a | . | c | d |
```

Comment:
@ekuns Were you able to find such an example with the script? :)

Reply:
Yes, except for changing the names, that output is taken directly from output when I run it against a few of the accounts I have access to.